pluginLauncher/lib/include/DobbyRdkPluginManager.h

```diff
@@ -87,8 +88,8 @@ class DobbyRdkPluginManager
     std::shared_ptr<rt_dobby_schema> mContainerConfig;
     const std::string mRootfsPath;
     const std::string mPluginPath;
-    const std::shared_ptr<DobbyRdkPluginUtils> mUtils;
```

Comment:
Make sure that if you didn't change anything lines are in the same place to avoid merge conflicts in future.